operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/ControllerConfiguration.java

```diff
@@ -97,4 +99,17 @@ default void setConfigurationService(ConfigurationService service) {}
   default boolean useFinalizer() {
     return !Controller.NO_FINALIZER.equals(getFinalizer());
   }
+
+  /**
+   * Allow controllers to filter events before they are provided to the
+   * {@link io.javaoperatorsdk.operator.processing.event.EventHandler}.
+   * </p>
+   * Note that the provided filter is combined with {@link #isGenerationAware()} to compute the
+   * final set of fiolters that should be applied;
+   *
+   * @return
+   */
+  default CustomResourceEventFilter<R> getEventFilter() {
```

Comment:
This should be a list too, or?

Reply:
It can be a list but the it will become an AND so the rationale here is that if you have to chain multiple filers, then you can probably chain them using the `and` / `or` operations provided by the interface so you are free to compose them as you like.

But no problem to make this a list.

Reply:
You right, is might be a too strong assumption to make. Thus that this would be usually an "AND" relationship in those filters.